From: Rene Kita <mail@rkta.de>
Date: Sat Sep 21 14:46:13 CEST 2024
Subject: Drop help file
Patch: patches/Drop-help-file.patch

Too many places to keep documentation up-to-date.
---
 w3mhelp-lynx_en.html.in |  141 -------------------------------------------
 w3mhelp-lynx_ja.html.in |  151 ----------------------------------------------
 w3mhelp-w3m_en.html.in  |  155 -----------------------------------------------
 w3mhelp-w3m_ja.html.in  |  156 ------------------------------------------------
 4 files changed, 603 deletions(-)

--- a/w3mhelp-lynx_en.html.in
+++ /dev/null
@@ -1,141 +0,0 @@
-<HTML>
-<HEAD>
-<TITLE>w3m help page</TITLE>
-</HEAD>
-<BODY>
-<CENTER>
-******* 
-<A HREF="http://w3m.sourceforge.net/">
-w3m</A>
- (WWW-wo-Miru)  Version @CURRENT_VERSION@ by 
-<A HREF="mailto:aito@fw.ipsj.or.jp">A.ITO</A> ********<BR>
-             ***** Key assign table *****
-</CENTER>
-
-<A HREF="w3mhelp-lynx_ja.html">Japanese</A>
-
-<H3>Page/Cursor motion</H3>
-<table cellpadding=0>
-<TR><TD WIDTH=100>SPC,C-v,+<TD>Forward page
-<TR><TD>b,ESC v,-<TD>Previous page
-<TR><TD>l<TD>Cursor right
-<TR><TD>h<TD>Cursor left
-<TR><TD>j<TD>Cursor down
-<TR><TD>k<TD>Cursor up
-<TR><TD>J<TD>Roll up one line
-<TR><TD>K<TD>Roll down one line
-<TR><TD>^<TD>Go to the beginning of line
-<TR><TD>$<TD>Go to the end of line
-<TR><TD>&gt;<TD>Shift screen right
-<TR><TD>&lt;<TD>Shift screen left
-<TR><TD>C-a<TD>Go to the first line
-<TR><TD>C-e<TD>Go to the last line
-<TR><TD>G<TD>Go to the specified line
-<TR><TD>Z<TD>Move to the center line
-<TR><TD>z<TD>Move to the center column
-<TR><TD>TAB,C-n,Down arrow<TD>Move to next hyperlink
-<TR><TD>ESC TAB,C-p,Up arrow<TD>Move to previous hyperlink
-<TR><TD>C-g<TD>Show current page position
-</table>
-
-
-<H2>Hyperlink operation</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>RET, C-f, Right arrow<TD>Follow hyperlink
-<TR><TD>d, ESC RET<TD>Save link to file
-<TR><TD>u<TD>Peek at link URL
-<TR><TD>i<TD>Peek at image URL
-<TR><TD>I<TD>View inline image
-<TR><TD>ESC I<TD>Save inline image to file
-<TR><TD>:<TD>Mark URL-like strings as links
-<TR><TD>ESC :<TD>Mark Message-ID-like strings as links
-<TR><TD>c<TD>Peek at current URL
-<TR><TD>=<TD>Display information about current document
-<TR><TD>C-h<TD>View browser history
-<TR><TD>F<TD>Render frames
-<TR><TD>M<TD>Browse current document using external browser
-(prefix 2, 3, ..., or 9 to invoke alternate configured browsers, e.g. 3 M)
-<TR><TD>ESC M<TD>Browse link using external browser
-(prefixed as above, e.g. 3ESC M)
-</table>
-
-<H2>File/Stream operation</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>g,U<TD>Open URL
-<TR><TD>V<TD>View new file
-<TR><TD>@<TD>Execute shell command and load
-<TR><TD>#<TD>Execute shell command and browse
-</table>
-
-<H2>Buffer operation</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>B, C-b, Left arrow<TD>Back to the previous buffer
-<TR><TD>\<TD>View HTML source
-<TR><TD>s<TD>Select buffer
-<TR><TD>E<TD>Edit buffer source
-<TR><TD>C-l, C-w<TD>Redraw screen
-<TR><TD>R, C-r<TD>Reload buffer
-<TR><TD>S, p<TD>Save buffer
-<TR><TD>ESC s<TD>Save source
-<TR><TD>ESC e<TD>Edit buffer image
-</table>
-
-<H2>Buffer selection mode</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>k, C-p<TD>Select previous buffer
-<TR><TD>j, C-n<TD>Select next buffer
-<TR><TD>D<TD>Delete current buffer
-<TR><TD>RET<TD>Go to the selected buffer
-</table>
-
-<H2>Bookmark operation</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>v, ESC b<TD>Load bookmark
-<TR><TD>a, ESC a<TD>Add current to bookmark
-</table>
-
-<H2>Search</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>/, C-s<TD>Search forward
-<TR><TD>n<TD>Search next
-<TR><TD>w<TD>Toggle search wrap mode
-</table>
-
-<H2>Mark operation</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>C-SPC<TD>Set/unset mark
-<TR><TD>P<TD>Go to previous mark
-<TR><TD>N<TD>Go to next mark
-<TR><TD>"<TD>Mark by regular expression
-</table>
-
-<H2>Miscellany</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>!<TD>Execute shell command
-<TR><TD>H, ?<TD>Help (load this file)
-<TR><TD>o<TD>Set option
-<TR><TD>C-k<TD>Show cookie jar
-<TR><TD>C-c<TD>Stop
-<TR><TD>C-z<TD>Suspend
-<TR><TD>q<TD>Quit (with confirmation, if you like)
-<TR><TD>Q<TD>Quit without confirmation
-</table>
-
-<H2>Line-editing mode</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>C-f<TD>Move cursor forward
-<TR><TD>C-b<TD>Move cursor backward
-<TR><TD>C-h<TD>Delete previous character
-<TR><TD>C-d<TD>Delete current character
-<TR><TD>C-k<TD>Kill everything after cursor
-<TR><TD>C-u<TD>Kill everything before cursor
-<TR><TD>C-a<TD>Move to the top of line
-<TR><TD>C-e<TD>Move to the bottom of line
-<TR><TD>C-p<TD>Fetch the previous string from the history list
-<TR><TD>C-n<TD>Fetch the next string from the history list
-<TR><TD>TAB,SPC<TD>Complete filename
-<TR><TD>RETURN<TD>Accept
-</table>
-<HR>
-</body>
-</html>
--- a/w3mhelp-lynx_ja.html.in
+++ /dev/null
@@ -1,151 +0,0 @@
-<HTML>
-<HEAD>
-<TITLE>w3m ヘルプページ</TITLE>
-</HEAD>
-
-<BODY>
-<CENTER>
- ******* 
- <A HREF="http://w3m.sourceforge.net/">w3m</A>
- (WWW-wo-Miru) Version @CURRENT_VERSION@ by
- <A HREF="mailto:aito@fw.ipsj.or.jp">A.ITO(伊藤彰則)</A>
- ********
- <BR>
- ***** lynx-like キー割り当て *****
-</CENTER>
-
-<A HREF="w3mhelp-lynx_en.html">English</A>
-
-<H3>ページ/カーソル移動</H3>
-<table cellpadding=0>
-<TR><TD>SPC,C-v,+<TD>次のページを表示します．
-<TR><TD>b,ESC v,-<TD>前のページを表示します．
-<TR><TD>l<TD>カーソルを右に移動します．
-<TR><TD>h<TD>カーソルを左に移動します．
-<TR><TD>j<TD>カーソルを下に移動します．
-<TR><TD>k<TD>カーソルを上に移動します．
-<TR><TD>J<TD>画面を1行上にスクロールします．
-<TR><TD>K<TD>画面を1行下にスクロールします．
-<TR><TD>^<TD>行頭に移動します．
-<TR><TD>$<TD>行末に移動します．
-<TR><TD>&gt;<TD>画面全体を右にずらします．(表示内容を左にずらす)
-<TR><TD>&lt;<TD>画面全体を左にずらします．(表示内容を右にずらす)
-<TR><TD>C-a<TD>文書のいちばん上の行に移動します．
-<TR><TD>C-e<TD>文書のいちばん下の行に移動します．
-<TR><TD>G<TD>画面下で行番号を入力し，そこで指定した行に移動します．
-ここで $ を入力すると，最終行に移動します．
-<TR><TD>Z<TD>カーソルのある位置を行の中央に移動します．
-<TR><TD>z<TD>カーソルのある行を画面の中央に移動します．
-<TR><TD>TAB, C-n, 下矢印<TD>次のリンクに移動します．
-<TR><TD>ESC TAB, C-p, 上矢印<TD>前のリンクに移動します．
-<TR><TD>C-g<TD>ページ中での現在位置を表示します．
-</table>
-
-<H3>ハイパーリンク操作</H3>
-<table cellpadding=0>
-<TR><TD WIDTH=100>RET, C-f, 右矢印<TD>現在カーソルがあるリンクが指す先の文書を読みこみます．
-<TR><TD>d, ESC RET<TD>現在カーソルがあるリンクが指す先の文書をファイルに保存します．
-<TR><TD>u<TD>現在カーソルがあるリンクが指す先のURLを表示します．
-<TR><TD>i<TD>リンクに関連付けられた画像へのURLを表示します．
-<TR><TD>I<TD>現在カーソルがあるリンクに対応する画像を表示します．
-<TR><TD>ESC I<TD>現在カーソルがあるリンクが指す画像をファイルに保存します．
-<TR><TD>:<TD>URL風の文字列をリンクにします．この機能は，HTMLでない文書を
-読んでいるときにも有効です．
-<TR><TD>ESC :<TD>Message-ID風の文字列を，news: のリンクにします．この機能は，HTMLでない文書を読んでいるときにも有効です．
-<TR><TD>c<TD>現在の文書のURLを表示します．
-<TR><TD>=<TD>現在の文書に関する情報を表示します．
-<TR><TD>C-h<TD>URL履歴を表示します．
-<TR><TD>F<TD>&lt;FRAMESET&gt;を含む文書を表示しているときに，&lt;FRAME&gt;
-タグの指す複数の文書を1つの文書に変換して表示します．
-<TR><TD>M<TD>現在見ているページを，外部ブラウザを使って表示します．
-2M, 3M, ..., 9M で2番目，3番目，9番目のブラウザを使います．
-<TR><TD>ESC M<TD>現在のリンク先を，外部ブラウザを使って表示します．
-2ESC M, 3ESC M, ..., 9ESC M で2番目，3番目，9番目のブラウザを使います．
-</table>
-
-<H3>ファイルとURL関係の操作</H3>
-<table cellpadding=0>
-<TR><TD WIDTH=100>g, U<TD>URLを指定して開きます．
-<TR><TD>V<TD>ローカルファイルを指定して開きます．
-<TR><TD>@<TD>コマンドを実行し，結果を全部読んでから表示します．
-<TR><TD>#<TD>コマンドを実行し，結果を読みこみながら表示します．
-</table>
-
-<H3>バッファ操作</H3>
-<table cellpadding=0>
-<TR><TD WIDTH=100>B, C-b, 左矢印<TD>現在見ているバッファを削除し，一つ前のバッファを表示します．
-<TR><TD>\<TD>HTMLのソースを表示します．
-<TR><TD>s<TD>バッファ選択モードに入ります．
-<TR><TD>E<TD>現在見ているバッファがローカルファイルの場合，そのファイルをエディタで編集します．エディタを終了した後，そのファイルを再度読み込みます．
-<TR><TD>C-l, C-w<TD>画面を再描画します．
-<TR><TD>R, C-r<TD>バッファを再度読み込みます．
-<TR><TD>S, p<TD>バッファの表示内容をファイルに保存します．
-<TR><TD>ESC s<TD>HTMLのソースをファイルに保存します．v でソースを表示して S で
-保存するのとほぼ同じですが，ESC s で保存したファイルは漢字コードがオリジナルの
-ままであるのに対して，v S で保存すると現在表示に使っている漢字コードに変換され
-て保存されます．
-<TR><TD>ESC e<TD>現在表示されているバッファを，表示されている形式のまま
-エディタで編集します．
-</table>
-
-<H3>バッファ選択モード</H3>
-"s" でバッファ選択モードに入ったときのキー操作です．
-<table cellpadding=0>
-<TR><TD WIDTH=100>k,C-p<TD>一つ上のバッファを選択します．
-<TR><TD>j,C-n<TD>一つ下のバッファを選択します．
-<TR><TD>D<TD>現在選択しているバッファを削除します．
-<TR><TD>RET<TD>現在選択しているバッファを表示します．
-</table>
-
-<H3>ブックマーク操作</H3>
-<table cellpadding=0>
-<TR><TD WIDTH=100>v, ESC b<TD>ブックマークを読み込みます．
-<TR><TD>a, ESC a<TD>現在見ているページをブックマークに追加します．
-</table>
-
-<H3>検索</H3>
-<table cellpadding=0>
-<TR><TD WIDTH=100>/,C-s<TD>現在のカーソル位置からファイル末尾に向かって正規表現を検索します．
-<TR><TD>n<TD>次を検索します．
-<TR><TD>w<TD>折り返し検索モードを切り換えます．
-</table>
-
-<H3>マーク操作</H3>
-<table cellpadding=0>
-<TR><TD WIDTH=100>C-SPC<TD>マークを設定／解除します．マークは反転表示されます．
-<TR><TD>P<TD>一つ前のマークに移動します．
-<TR><TD>N<TD>一つ後のマークに移動します．
-<TR><TD>"<TD>正規表現で指定された文字列を全てマークします．
-</table>
-
-<H3>その他</H3>
-<table cellpadding=0>
-<TR><TD WIDTH=100>!<TD>シェルコマンドを実行します．
-<TR><TD>H, ?<TD>ヘルプファイルを表示します．
-<TR><TD>o<TD>オプション設定パネルを表示します．
-<TR><TD>C-k</TD> <TD>クッキー一覧を表示</TD></TR>
-<TR><TD>C-c<TD>文書の読み込みを中断します．
-<TR><TD>C-z</TD><TD>サスペンドします．</TD></TR>
-<TR><TD>q<TD>w3mを終了します．オプションの設定によって，終了するかどうか確認します．
-<TR><TD>Q<TD>確認せずにw3mを終了します．
-</table>
-
-<H3>行編集モード</H3>
-画面の最下行で文字列を入力する場合に有効なキー操作です．
-<table cellpadding=0>
-<TR><TD WIDTH=100>C-f<TD>カーソルを右に移動します．
-<TR><TD>C-b<TD>カーソルを左に移動します．
-<TR><TD>C-h<TD>カーソルの直前の文字を削除します．
-<TR><TD>C-d<TD>カーソル位置の文字を削除します．
-<TR><TD>C-k<TD>カーソル位置から後を削除します．
-<TR><TD>C-u<TD>カーソル位置から前を削除します．
-<TR><TD>C-a<TD>文字列の先頭に移動します．
-<TR><TD>C-e<TD>文字列の最後に移動します．
-<TR><TD>SPC<TD>ファイル名入力時に，ファイル名を補完します．
-<TR><TD>RETURN<TD>入力を終了します．
-</table>
-
-
-
-</body>
-</html>
--- a/w3mhelp-w3m_en.html.in
+++ /dev/null
@@ -1,155 +0,0 @@
-<HTML>
-<HEAD>
-<TITLE>w3m help page</TITLE>
-</HEAD>
-<BODY>
-<CENTER>
-******* 
-<A HREF="http://w3m.sourceforge.net/">
-w3m</A>
- (WWW-wo-Miru)  Version @CURRENT_VERSION@ by 
-<A HREF="mailto:aito@fw.ipsj.or.jp">A.ITO</A> ********<BR>
-             ***** Key assign table *****
-</CENTER>
-
-<A HREF="w3mhelp-w3m_ja.html">Japanese</A>
-
-<H2>Page/Cursor motion</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>SPC,C-v<TD>Forward page
-<TR><TD>b,ESC v<TD>Backward page
-<TR><TD>l,C-f<TD>Cursor right
-<TR><TD>h,C-b<TD>Cursor left
-<TR><TD>j,C-n<TD>Cursor down
-<TR><TD>k,C-p<TD>Cursor up
-<TR><TD>J<TD>Roll up one line
-<TR><TD>K<TD>Roll down one line
-<TR><TD>^,C-a<TD>Go to the beginning of line
-<TR><TD>$,C-e<TD>Go to the end of line
-<TR><TD>w<TD>Go to next word
-<TR><TD>W<TD>Go to previous word
-<TR><TD>&gt;<TD>Shift screen right
-<TR><TD>&lt;<TD>Shift screen left
-<TR><TD>.<TD>Shift screen one column right 
-<TR><TD>,<TD>Shift screen one column left
-<TR><TD>g,M-&lt;<TD>Go to the first line
-<TR><TD>G,M-&gt;<TD>Go to the last line
-<TR><TD>ESC g<TD>Go to specified line
-<TR><TD>Z<TD>Move to the center line
-<TR><TD>z<TD>Move to the center column
-<TR><TD>TAB<TD>Move to next hyperlink
-<TR><TD>C-u,ESC TAB<TD>Move to previous hyperlink
-<TR><TD>[<TD>Go to the first link
-<TR><TD>]<TD>Go to the last link
-</table>
-
-<H2>Hyperlink operation</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>RET<TD>Follow hyperlink
-<TR><TD>a, ESC RET<TD>Save link to file
-<TR><TD>u<TD>Peek at link URL
-<TR><TD>i<TD>Peek at image URL
-<TR><TD>I<TD>View inline image
-<TR><TD>ESC I<TD>Save inline image to file
-<TR><TD>:<TD>Mark URL-like strings as links
-<TR><TD>ESC :<TD>Mark Message-ID-like strings as links
-<TR><TD>c<TD>Peek at current URL
-<TR><TD>=<TD>Display information about current document
-<TR><TD>C-g<TD>Show current line number
-<TR><TD>C-h<TD>View browser history
-<TR><TD>F<TD>Render frames
-<TR><TD>M<TD>Browse current document using external browser
-(prefix 2, 3, ..., or 9 to invoke alternate configured browsers, e.g. 3 M)
-<TR><TD>ESC M<TD>Browse link using external browser
-(prefixed as above, e.g. 3ESC M)
-</table>
-
-<H2>File/Stream operation</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>U<TD>Open URL
-<TR><TD>V<TD>View new file
-<TR><TD>@<TD>Execute shell command and load
-<TR><TD>#<TD>Execute shell command and browse
-</table>
-
-<H2>Buffer operation</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>B<TD>Back to the previous buffer
-<TR><TD>v<TD>View HTML source
-<TR><TD>s<TD>Select buffer
-<TR><TD>E<TD>Edit buffer source
-<TR><TD>C-l<TD>Redraw screen
-<TR><TD>R<TD>Reload buffer
-<TR><TD>S<TD>Save buffer
-<TR><TD>ESC s<TD>Save source
-<TR><TD>ESC e<TD>Edit buffer image
-</table>
-
-<H2>Buffer selection mode</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>k, C-p<TD>Select previous buffer
-<TR><TD>j, C-n<TD>Select next buffer
-<TR><TD>D<TD>Delete current buffer
-<TR><TD>RET<TD>Go to the selected buffer
-</table>
-
-<H2>Bookmark operation</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>ESC b<TD>Load bookmark
-<TR><TD>ESC a<TD>Add current to bookmark
-</table>
-
-<H2>Search</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>/,C-s<TD>Search forward
-<TR><TD>?,C-r<TD>Search backward
-<TR><TD>n<TD>Search next
-<TR><TD>N<TD>Search previous
-<TR><TD>C-w<TD>Toggle search wrap mode
-</table>
-
-<H2>Dictionary look-up</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>M-w<TD>Execute dictionary command (see README.dict)
-<TR><TD>M-W<TD>Execute dictionary command for word at cursor
-</table>
-
-<H2>Mark operation</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>C-SPC<TD>Set/unset mark
-<TR><TD>ESC p<TD>Go to previous mark
-<TR><TD>ESC n<TD>Go to next mark
-<TR><TD>"<TD>Mark by regular expression
-</table>
-
-<H2>Miscellany</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>!<TD>Execute shell command
-<TR><TD>H<TD>Help (load this file)
-<TR><TD>o<TD>Set option
-<TR><TD>C-k<TD>Show cookie jar
-<TR><TD>C-c<TD>Stop
-<TR><TD>C-z<TD>Suspend
-<TR><TD>q<TD>Quit (with confirmation, if you like)
-<TR><TD>Q<TD>Quit without confirmation
-</table>
-
-<H2>Line-editing mode</H2>
-<table cellpadding=0>
-<TR><TD WIDTH=100>C-f<TD>Move cursor forward
-<TR><TD>C-b<TD>Move cursor backward
-<TR><TD>C-h<TD>Delete previous character
-<TR><TD>C-d<TD>Delete current character
-<TR><TD>C-k<TD>Kill everything after cursor
-<TR><TD>C-u<TD>Kill everything before cursor
-<TR><TD>C-a<TD>Move to the top of line
-<TR><TD>C-e<TD>Move to the bottom of line
-<TR><TD>C-p<TD>Fetch the previous string from the history list
-<TR><TD>C-n<TD>Fetch the next string from the history list
-<TR><TD>TAB,SPC<TD>Complete filename
-<TR><TD>RETURN<TD>Accept
-</table>
-<HR>
-</BODY>
-</HTML>
-
--- a/w3mhelp-w3m_ja.html.in
+++ /dev/null
@@ -1,156 +0,0 @@
-<HTML>
-<HEAD>
-<TITLE>w3m ヘルプ ページ</TITLE>
-</HEAD>
-
-<BODY>
-<CENTER>
- ******* 
- <A HREF="http://w3m.sourceforge.net/">w3m</A>
- (WWW-wo-Miru) Version @CURRENT_VERSION@ by
- <A HREF="mailto:aito@fw.ipsj.or.jp">A.ITO(伊藤彰則)</A>
- ********
- <BR>
- ***** キー割り当て *****
-</CENTER>
-
-<A HREF="w3mhelp-w3m_en.html">English</A>
-
-<H2>ページ/カーソル移動</H2>
-<TABLE CELLPADDING=0>
- <TR> <TD WIDTH=100>SPC,C-v</TD> <TD>次ページ</TD></TR>
- <TR> <TD>b,ESC v</TD> <TD>前ページ</TD></TR>
- <TR> <TD>l,C-f</TD> <TD>カーソルを右へ移動</TD></TR>
- <TR> <TD>h,C-b</TD> <TD>カーソルを左へ移動</TD></TR>
- <TR> <TD>j,C-n</TD> <TD>カーソルを下へ移動</TD></TR>
- <TR> <TD>k,C-p</TD> <TD>カーソルを上へ移動</TD></TR>
- <TR> <TD>J</TD><TD>一行上にスクロール(進む)</TD></TR>
- <TR> <TD>K</TD><TD>一行下にスクロール(戻る)</TD></TR>
- <TR> <TD>^,C-a</TD><TD>行頭に移動</TD></TR>
- <TR> <TD>$,C-e</TD><TD>行末に移動</TD></TR>
- <TR> <TD>w</TD><TD>次の単語に移動</TD></TR>
- <TR> <TD>W</TD><TD>前の単語に移動</TD></TR>
- <TR> <TD>&gt;</TD> <TD>右に一画面分シフト</TD></TR>
- <TR> <TD>&lt;</TD> <TD>左に一画面分シフト</TD></TR>
- <TR> <TD>.</TD> <TD>右に一文字分シフト</TD></TR>
- <TR> <TD>,</TD> <TD>左に一文字分シフト</TD></TR>
- <TR> <TD>g,M-&lt;</TD> <TD>ページの先頭行に移動</TD></TR>
- <TR> <TD>G,M-&gt;</TD> <TD>ページの最終行に移動</TD></TR>
- <TR> <TD>ESC g</TD> <TD>指定行に移動</TD></TR>
- <TR> <TD>Z</TD> <TD>カーソルのある位置を行の中央に移動</TD></TR>
- <TR> <TD>z</TD> <TD>カーソルのある行を画面の中央に移動</TD></TR>
- <TR> <TD>TAB</TD> <TD>次のハイパーリンクに移動</TD></TR>
- <TR> <TD>C-u, ESC TAB</TD> <TD>前のハイパーリンクに移動</TD></TR>
- <TR> <TD>[</TD> <TD>最初のハイパーリンクに移動</TD></TR>
- <TR> <TD>]</TD> <TD>最後のハイパーリンクに移動</TD></TR>
-</TABLE>
-
-<H2>ハイパーリンク操作</H2>
-
-<TABLE CELLPADDING=0>
- <TR> <TD WIDTH=100>RET</TD> <TD>カーソル下のリンクへ飛ぶ</TD></TR>
- <TR> <TD>a, ESC RET</TD> <TD>リンク先の文書をファイルに保存</TD></TR>
- <TR> <TD>ESC I</TD> <TD>リンク先の画像をファイルに保存</TD></TR>
- <TR> <TD>u</TD> <TD>リンク先URLの表示</TD></TR>
- <TR> <TD>i</TD> <TD>リンクに関連付けられた画像へのURLの表示</TD></TR>
- <TR> <TD>I</TD> <TD>インラインイメージの表示</TD></TR>
- <TR> <TD>:</TD> <TD>URLのような文字をアンカーとしてマーク</TD></TR>
- <TR> <TD>ESC :</TD> <TD>Message-IDのような文字をアンカーとしてマーク</TD></TR>
- <TR> <TD>c</TD> <TD>現ページのURLを表示</TD></TR>
- <TR> <TD>=</TD> <TD>現ドキュメントの情報を表示</TD></TR>
- <TR> <TD>C-g</TD> <TD>ページ中での現在位置を表示</TD></TR>
- <TR> <TD>C-h</TD> <TD>URL履歴を表示</TD></TR>
- <TR> <TD>F</TD> <TD>フレームを表示する</TD></TR>
- <TR> <TD>M</TD> <TD>現ページを外部ブラウザで表示する(2M, 3M, ..., 9M で2番目，3番目，9番目のブラウザを呼びだす)</TD></TR>
- <TR> <TD>ESC M</TD> <TD>リンク先を外部ブラウザで表示する(2ESC M, 3ESC M, ..., 9ESC M で2番目，3番目，9番目のブラウザを呼びだす)</TD></TR>
-</TABLE>
-
-<H2>ファイル/ストリーム操作</H2>
-<TABLE CELLPADDING=0>
- <TR> <TD WIDTH=100>U</TD> <TD>URLを指定</TD></TR>
- <TR> <TD>V</TD> <TD>ファイルを開く</TD></TR>
- <TR> <TD>@</TD> <TD>シェルを起動し読み込む</TD></TR>
- <TR> <TD>#</TD> <TD>シェルを起動し読み込む</TD></TR>
-</TABLE>
-
-<H2>バッファ操作</H2>
-<TABLE CELLPADDING=0>
- <TR> <TD WIDTH=100>B</TD> <TD>前のバッファに移動</TD></TR>
- <TR> <TD>v</TD> <TD>HTMLソースを表示</TD></TR>
- <TR> <TD>s</TD> <TD>バッファを選択</TD></TR>
- <TR> <TD>E</TD> <TD>バッファのソースを編集</TD></TR>
- <TR> <TD>C-l</TD> <TD>画面の再描画</TD></TR>
- <TR> <TD>R</TD> <TD>バッファを再読み込み</TD></TR>
- <TR> <TD>S</TD> <TD>バッファを保存</TD></TR>
- <TR> <TD>ESC s</TD> <TD>HTMLソースを保存</TD></TR>
- <TR> <TD>ESC e</TD> <TD>バッファの表示イメージを編集</TD></TR>
-</TABLE>
-
-<H2>バッファ選択モード</H2>
-"s" でバッファ選択モードに入ったときのキー操作
-<TABLE CELLPADDING=0>
- <TR> <TD WIDTH=100>k,C-p</TD> <TD>一つ上のバッファを選択</TD></TR>
- <TR> <TD>j,C-n</TD> <TD>一つ下のバッファを選択</TD></TR>
- <TR> <TD>D</TD> <TD>現在選択しているバッファを削除</TD></TR>
- <TR> <TD>RET</TD> <TD>現在選択しているバッファを表示</TD></TR>
-</TABLE>
-
-<H2>ブックマーク操作</H2>
-<TABLE CELLPADDING=0>
- <TR> <TD WIDTH=100>ESC b</TD> <TD>ブックマークを読み込む</TD></TR>
- <TR> <TD>ESC a</TD> <TD>現ページをブックマークに追加</TD></TR>
-</TABLE>
-
-<H2>検索</H2>
-<TABLE CELLPADDING=0>
- <TR> <TD WIDTH=100>/,C-s</TD> <TD>前方検索</TD></TR>
- <TR> <TD>?,C-r</TD> <TD>後方検索</TD></TR>
- <TR> <TD>n</TD> <TD>次を検索</TD></TR>
- <TR> <TD>N</TD> <TD>前を検索</TD></TR>
- <TR> <TD>C-w</TD> <TD>折り返し検索モードの切り換え</TD></TR>
-</TABLE>
-
-<H2>辞書検索</H2>
-<TABLE CELLPADDING=0>
- <TR> <TD WIDTH=100>M-w</TD> <TD>入力した単語を辞書コマンドで検索</TD></TR>
- <TR> <TD>M-W</TD> <TD>カーソル位置の単語を辞書コマンドで検索</TD></TR>
-</TABLE>
-
-<H2>マーク操作</H2>
-<TABLE CELLPADDING=0>
- <TR> <TD WIDTH=100>C-SPC</TD> <TD>マークの設定/消去</TD></TR>
- <TR> <TD>ESC p</TD> <TD>前のマークへ移動</TD></TR>
- <TR> <TD>ESC n</TD> <TD>次のマークへ移動</TD></TR>
- <TR> <TD>"</TD> <TD>正規表現によるマーク</TD></TR>
-</TABLE>
-
-<H2>その他</H2>
-<TABLE CELLPADDING=0>
- <TR> <TD WIDTH=100>!</TD> <TD>シェルの実行</TD></TR>
- <TR> <TD>H</TD> <TD>ヘルプ(このファイルを表示)</TD></TR>
- <TR> <TD>o</TD> <TD>オプション設定</TD></TR>
- <TR> <TD>C-k</TD> <TD>クッキー一覧を表示</TD></TR>
- <TR> <TD>C-c</TD> <TD>文書の読み込みを中断</TD></TR>
- <TR> <TD>C-z</TD> <TD>サスペンド</TD></TR>
- <TR> <TD>q</TD> <TD>w3mを終了(確認あり)</TD></TR>
- <TR> <TD>Q</TD> <TD>w3mを終了(確認なし)</TD></TR>
-</TABLE>
-
-<H2>行編集モード</H2>
-<TABLE CELLPADDING=0>
- <TR> <TD WIDTH=100>C-f</TD> <TD>カーソルを右へ移動</TD></TR>
- <TR> <TD>C-b</TD> <TD>カーソルを左へ移動</TD></TR>
- <TR> <TD>C-h</TD> <TD>前の文字を削除</TD></TR>
- <TR> <TD>C-d</TD> <TD>カーソル位置の文字を削除</TD></TR>
- <TR> <TD>C-k</TD> <TD>カーソルの後方を全て削除</TD></TR>
- <TR> <TD>C-u</TD> <TD>カーソルの前までを全て削除</TD></TR>
- <TR> <TD>C-a</TD> <TD>行頭に移動</TD></TR>
- <TR> <TD>C-e</TD> <TD>行末へ移動</TD></TR>
- <TR> <TD>C-p</TD> <TD>ヒストリから一つ前の文字列を取り出す</TD></TR>
- <TR> <TD>C-n</TD> <TD>ヒストリから次の文字列を取り出す</TD></TR>
- <TR> <TD>TAB,SPC</TD> <TD>ファイル名の補完</TD></TR>
- <TR> <TD>RETURN</TD> <TD>入力終了</TD></TR>
-</TABLE>
-<HR>
-</BODY>
-</HTML>
